great_expectations/rule_based_profiler/config/base.py

```diff
@@ -181,8 +181,8 @@ def __init__(
         self,
         name: str,
         domain_builder: DomainBuilderConfig,
-        parameter_builders: List[ParameterBuilderConfig],
         expectation_configuration_builders: List[ExpectationConfigurationBuilderConfig],
+        parameter_builders: Optional[List[ParameterBuilderConfig]] = None,
```

Comment:
Per discussion with @alexsherstinsky , `parameter_builders` should be optional. This is reflected in the schema but should also impact the config object.